lib/reversetunnel/localsite.go

```diff
@@ -164,27 +166,18 @@ func (s *localSite) GetLastConnected() time.Time {
 	return s.clock.Now()
 }
 
-func (s *localSite) DialAuthServer() (conn net.Conn, err error) {
-	// get list of local auth servers
-	authServers, err := s.client.GetAuthServers()
-	if err != nil {
-		return nil, trace.Wrap(err)
-	}
-
-	if len(authServers) < 1 {
+func (s *localSite) DialAuthServer() (net.Conn, error) {
+	if len(s.authServers) == 0 {
 		return nil, trace.ConnectionProblem(nil, "no auth servers available")
 	}
 
-	// try and dial to one of them, as soon as we are successful, return the net.Conn
-	for _, authServer := range authServers {
```

Comment:
Are there any implications in dropping this loop that we should be aware of?

Reply:
All our current clients that dial `@remote-auth-addr` through a reverse tunnel connection have builtin retries, so even in the case of multiple `auth_servers` specified in the configuration file (which is very much discouraged) this is going to be fine.